apps/core/scenes/bookingDetail/menuGroups/Manage.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import * as React from 'react';
+import { Platform } from 'react-native';
+import {
+  MenuGroup,
+  MenuItem,
+  Icon,
+  StyleSheet,
+} from '@kiwicom/universal-components';
+import {
+  withNavigation,
+  type Navigation,
+  Routes,
+} from '@kiwicom/margarita-navigation';
+
+type Props = {|
+  +navigation: Navigation,
+|};
+
+class ManageMenuGroup extends React.Component<Props> {
+  onPressInvite = () => {
+    this.props.navigation.navigate(Routes.MANAGE_SHARE_BOOKING);
+  };
+
+  onPressHelp = () => {
+    this.props.navigation.navigate(Routes.MANAGE_HELP);
+  };
+
+  onPressOther = () => {
+    this.props.navigation.navigate(Routes.MANAGE_OTHER);
+  };
+
+  render() {
+    let shareIcon;
+    switch (Platform.OS) {
+      case 'ios': {
+        shareIcon = 'share-ios';
+        break;
+      }
+      case 'android': {
+        shareIcon = 'share-android';
+        break;
+      }
+      default: {
+        shareIcon = 'share';
+        break;
+      }
+    }
```

Comment:
How about creating a ShareIcon component ? We will use this logic also in boarding pass component